composer.json

```diff
@@ -24,6 +24,8 @@
         "php": "^7.3 || ^8.0",
         "cocur/slugify": "^3.0 || ^4.0",
         "doctrine/persistence": "^1.3.6 || ^2.0",
+        "knplabs/knp-menu": "^3.0",
```

Comment:
Do you need this one?

Reply:
https://github.com/sonata-project/SonataClassificationBundle/blob/master/src/Resources/views/CategoryAdmin/list_tab_menu.html.twig#L1

We're using  knp_menu_render which is define in knp-menu, so it seems better to add it to the require.